src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpRawlsDAO.scala

```diff
@@ -162,6 +165,7 @@ class HttpRawlsDAO(implicit val system: ActorSystem, implicit val materializer:
     val url = editBillingMembershipURL(projectId, role, email)
 
     userAuthedRequest(Put(url), true) flatMap { resp =>
+      resp.discardEntityBytes()
```

Comment:
should this be inside the `if (resp.status.isSuccess) {` clause? Does the `FCErrorReport(resp).flatMap { errorReport =>`  properly consume the entity?

Reply:
I probably shouldn't have speculated, I will verify this now (along with the other unclear bits)

Reply:
im a bit confused by this. while testing, i'm forcing it to take the FCErrorReport code path, and it works with or without discarding the bytes manually. is this the same thing we were talking about yesterday with the `logRequests` directive?